reference/features/distinct.md

```diff
@@ -1,16 +1,16 @@
 # Distinct attribute
 
-The **value** of a field whose attribute is set as a distinct attribute will always be **unique** in the returned documents.
+A distinct attribute is a field whose value is shared by various documents. It is most commonly used to prevent MeiliSearch from returning several instances of similar documents.
```

Comment:
```suggestion
The distinct attribute is a special, user-designated field. It is most commonly used to prevent MeiliSearch from returning a set of several similar documents, instead forcing it to return only one.
```